simulation_ws/src/hello_world_simulation/package.xml

```diff
@@ -11,8 +11,7 @@
   <buildtool_depend>catkin</buildtool_depend>
   <depend>gazebo_ros</depend>
   <depend>gazebo_plugins</depend>
+  <!-- Required to install .rviz model -->
   <depend>turtlebot3_description</depend>
-  <depend>turtlebot3_gazebo</depend>
-  <depend>turtlebot3_simulations</depend>
```

Comment:
We had `turtlebot3_simulations` because of `robot_state_publisher`. I added that dependency in the package `turtlebot3-description-reduced-mesh` because that package requires `robot_state_publisher`. Therefore, it's safe to remove them here.